### PR TITLE
[zh-hans] Add Ecto lessons about basic

### PR DESCRIPTION
**Pull request by caicaishmily**

### CODE REVIEW

File: zh-hans/lessons/specifics/ecto/basics.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+* MySQL
+* SQLite
+
+在本课中，我们将使用 PostgreSQL 适配器来配置 Ecto。
+
+### 开始
+
+在本课程中，我们将介绍有关 Ecto 的以下三部分：
+* Repository - 提供数据库的接口，包括连接
+* Migrations - 一种创建，修改和销毁数据库表和索引的机制
```
Comment (zillou) [inline zh-hans/lessons/specifics/ecto/basics.md]:
```suggestion
* Migrations - 一种创建，修改和删除数据库表和索引的机制
```

File: zh-hans/lessons/specifics/ecto/basics.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+  use Ecto.Schema
+
+  schema "people" do
+    field :name, :string
+    field :age, :integer, default: 0
+  end
+end
+```
+
+在这里我们可以看到 `Example.Person` 模块告诉 Ecto 这个 schema 与 `people` 表有关，我们有两列：一个字符串类型的 `name` 列和一个默认值为整数 `0` 的 `age` 列。
```
Comment (zillou) [inline zh-hans/lessons/specifics/ecto/basics.md]:
```suggestion
在这里我们可以看到 `Example.Person` 模块告诉 Ecto 这个 schema 与 `people` 表有关，我们有两个字段：一个字符串类型的 `name` 字段和一个整数类型的 `age` 字段，并指定 `age` 的默认值微 0。
```

Comment (zillou) [reply zh-hans/lessons/specifics/ecto/basics.md]: 我们一般把 column 叫字段吧？

**Comment by caicaishmily** [comment]
Thanks, I just updated the file path to right place. @zillou

**Comment by zillou** [comment]
Thank you very much, @caicaishmily .

**Pull request merged**